execution/evm/vm.go

```diff
@@ -493,6 +493,52 @@ func (vm *VM) call(caller acm.Account, callee acm.MutableAccount, code, input []
 			stack.Push64(int64(res))
 			vm.Debugf(" => 0x%X\n", res)
 
+		case SHL: //0x1B
+			shift, value := stack.PopBigInt(), stack.PopBigInt()
+
+			if shift.Cmp(Big256()) >= 0 {
+				reset := big.NewInt(0)
+				stack.PushBigInt(reset)
+				vm.Debugf(" %X << %X = %X\n", value, shift, reset)
+			} else {
+				shiftedValue := value.Lsh(value, uint(shift.Uint64()))
+				stack.PushBigInt(shiftedValue)
+				vm.Debugf(" %X << %X = %X\n", value, shift, shiftedValue)
```

Comment:
We would like to show the integer value rather than the hex here like in other arithemetic methods so these `%X`s shoiuld be `%v`s